pkg/sink/sink_test.go

```diff
@@ -212,13 +209,12 @@ func TestHandleEvent(t *testing.T) {
 				},
 			},
 		}
-		action := ktesting.NewCreateAction(pipelineResourceGVR, "foo", test.ToUnstructured(t, wantResource))
-		wantActions = append(wantActions, action)
+		wantPrs = append(wantPrs, wantResource)
```

Comment:
@ncskier  Thoughts on comparing the actions vs comparing the created/wanted resources directly?

Reply:
Comparing actions can sometimes be a little funky, because the order of the actions matters.

I like the idea of just comparing the created/wanted resources directly; it might lend itself to smaller, more focused tests and to table driven tests.

However, there might be more control by comparing actions?

Reply:
agreed...In all of the examples here though we are extract the resource from the action and then compare the resource so I'm not sure how much extra benefit we get from comparing actions directly...one nice thing I noticed is that the diffs are smaller now since its just the resources and not the entire action